applications/rovioli/src/datasource-rosbag.cc

```diff
@@ -45,7 +45,12 @@ DataSourceRosbag::DataSourceRosbag(
     : shutdown_requested_(false),
       all_data_streamed_(false),
       rosbag_path_filename_(rosbag_path_filename),
-      ros_topics_(ros_topics) {
+      ros_topics_(ros_topics),
+      last_imu_timestamp_ns_(aslam::time::getInvalidTime()) {
+  const uint32_t num_cameras = ros_topics_.camera_topic_cam_index_map.size();
```

Comment:
32 bits for `num_cameras` is quite optimistic. i don't think we will ever have 4 billion cameras ;)  i believe 8 bits are enough here